eZ/Publish/API/Repository/Tests/Stubs/RepositoryStub.php

```diff
@@ -632,6 +659,24 @@ public function rollback()
         }
 
         --$this->transactionDepth;
+        unset( $this->commitEventsQueue[$this->transactionCount] );
```

Comment:
Ouch :-S sorry @andrerom I don't have any clue about this, this is way too old for my memory.